charts/super-agent/templates/uninstall-job.yaml

```diff
@@ -0,0 +1,49 @@
+{{- if .Values.helm.create -}}
+{{- $installJobName := include "newrelic.common.naming.truncateToDNSWithSuffix" ( dict "name" (include "newrelic.common.naming.fullname" .) "suffix" "uninstall-job" ) -}}
+{{- $configMapName := include "newrelic.common.naming.truncateToDNSWithSuffix" ( dict "name" (include "newrelic.common.naming.fullname" .) "suffix" "manifests" ) -}}
```

Comment:
job-manifests instead of manifests?

Reply:
Done on [#b5da77e](https://github.com/newrelic/helm-charts/pull/1175/commits/b5da77e706e5cce2c1b77e36f72063bd23c1e715)